### PR TITLE
Requesting GitHub Codespaces for Node.js

### PR DESCRIPTION
**Pull request by ovflowd**
## GitHub Resource Request for Node.js

- **Estimated monthly spend:** USD 100.00
- **Requested Features:**
  - [ ] Actions
  - [x] Codespaces
  - [ ] Extra Large Runners (Actions)
- **Fund Request Champions:**
  - Claudio Wunder (@ovflowd) (CPC)
  - Rich Trott (@Trott) (TSC)

### Intended usage of the resources

GitHub Codespaces for Node.js will allow Node.js contributors to build, compile and run Node.js (and other repositories, such as Node.js Website) (it will allow Pull Request updates to be done from the web, code reviews from the web, and even contributors to commit changes or implement features).

The initial idea is that:

- Only a few repositories will be enabled
- Only a few members will have access from the beginning (it will be an opt-in process)
- We will monitor the first months for usage to understand how much we spend, if we need more, how sustainable this resource can be, and how many people are interested in it.

cc @openjs-foundation/cpc

### CODE REVIEW

**Comment by ovflowd** [comment]
cc @bensternthal

File: project-resources/GITHUB_REQUESTS/2023.md [hunk 1]
```diff
@@ -2,3 +2,4 @@
 
 First Name | Last Name | Project | Amount requested | Pull Request date | Pull Request link
 -|-|-|-|-|-
+Claudio | Wunder | Node.js | 100.00 USD | 08.16.2023 | https://github.com/openjs-foundation/cross-project-council/pulls/1149 |
```
Comment (ovflowd) [inline project-resources/GITHUB_REQUESTS/2023.md:5]: Yes I predicted what the Pull Request number would be lol

Comment (ljharb) [inline project-resources/GITHUB_REQUESTS/2023.md:5]:
```suggestion
Claudio | Wunder | Node.js | 100.00 USD | 2023.08.16 | https://github.com/openjs-foundation/cross-project-council/pulls/1149 |
```
so we don't confuse our european friends

**Review by ljharb — APPROVED** [review]

File: project-resources/GITHUB_REQUESTS/2023.md [hunk 2]
```diff
@@ -2,3 +2,4 @@
 
 First Name | Last Name | Project | Amount requested | Pull Request date | Pull Request link
 -|-|-|-|-|-
+Claudio | Wunder | Node.js | 100.00 USD | 2023.08.16 | https://github.com/openjs-foundation/cross-project-council/pulls/1149 |
```
Comment (tobie) [inline project-resources/GITHUB_REQUESTS/2023.md:5]:
```suggestion
Project  | Request | Cost / Period | Date | Champion | Pull Request link
-|-|-|-|-|-
Node.js | Codespaces | USD 100 / month | 2023.08.16 | Claudio Wunder | https://github.com/openjs-foundation/cross-project-council/pulls/1149 |
```

A few suggestions about the format of the requests in general (as this is the first one):

- I initially saw all of those zeroes and thought you were asking for a lot more; we don’t need to mention cents here.
- Knowing that this is a recurring expense is important. We could even consider a `frequency` column.
- `first name` / `last name` columns makes this look like a personal expense, so maybe we should use `project` / `champion` instead.
-  It's valuable to know what the request is for inline.

**Review by tobie — APPROVED** [review]

File: project-resources/GITHUB_REQUESTS/2023.md (hunk 2 above)
Comment (ovflowd) [reply project-resources/GITHUB_REQUESTS/2023.md:5]: This should probably be addressed on a separate PR 👀

Comment (tobie) [reply project-resources/GITHUB_REQUESTS/2023.md:5]: > This should probably be addressed on a separate PR 👀 (that should be merged before this one)

Whatever feels more convenient to you. I'm fine either way.

Comment (ovflowd) [reply project-resources/GITHUB_REQUESTS/2023.md:5]: Sure. Imma gonna proceed and open another PR for this + some changes to remove the cents mention on the template and also mention that these costs are intended to be recurring

Comment (ovflowd) [reply project-resources/GITHUB_REQUESTS/2023.md:5]: PR: https://github.com/openjs-foundation/cross-project-council/pull/1150

**Review by mcollina — APPROVED** [review]
lgtm

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by VoltrexKeyva — APPROVED** [review]

**Comment by ovflowd** [comment]
cc @openjs-foundation/cpc I've just updated this (can we get a re-review?)

**Review by tobie — APPROVED** [review]

**Comment by tobie** [comment]
What's the plan budget-wise, here, so we don't end up with the same issues as we did with the travel fund? /cc @bensternthal

**Comment by bensternthal** [comment]
@tobie for 2024 I expect us to have a separate budget line item for this type of expense.

**Comment by ovflowd** [comment]
I believe it is fine for us to merge this? @bensternthal can we get started on the process?

**Comment by bensternthal** [comment]
@ovflowd isn't the next step [here](https://github.com/openjs-foundation/cross-project-council/blob/main/project-resources/REQUESTING_GITHUB_RESOURCES.md#requesting-github-resources):  "If approved the CPC will discuss with OpenJS staff in the next CPC meeting"?

**Comment by ovflowd** [comment]
> @ovflowd isn't the next step [here](https://github.com/openjs-foundation/cross-project-council/blob/main/project-resources/REQUESTING_GITHUB_RESOURCES.md?rgh-link-date=2023-09-26T22%3A01%3A45Z#requesting-github-resources): "If approved the CPC will discuss with OpenJS staff in the next CPC meeting"?

Yup, so next meeting it is.

**Comment by ovflowd** [comment]
I forgot to bring this to the meeting, so adding this to the agenda for next meeting!

**Pull request merged**